spec/git_reflow/all-approval/git_reflow_spec.rb

```diff
@@ -18,6 +18,11 @@
   let(:existing_pull_request)  { GitReflow::GitServer::GitHub::PullRequest.new existing_pull_requests.first }
 
   before do
+
```

Comment:
@codenamev The original spec remains! It is just shifted into `all-approval` folder. I have modified the `git_reflow_spec.rb` file in the `2-lgtm` folder (similar tests to `all-approval`) to only include tests from `deliver` and removed all other tests for other commands. The spec with the full set of tests is in the `all-approval` folder. Please let me know if you want to move it back to the original location. Good suggestions!